lib/sentry/integrations/oban/error_reporter.ex

```diff
@@ -33,18 +33,22 @@ defmodule Sentry.Integrations.Oban.ErrorReporter do
       end
 
     _ =
-      Sentry.capture_exception(exception,
-        stacktrace: stacktrace,
-        tags: %{oban_worker: job.worker, oban_queue: job.queue, oban_state: job.state},
-        fingerprint: [
-          inspect(exception.__struct__),
-          inspect(job.worker),
-          Exception.message(exception)
-        ],
-        extra:
-          Map.take(job, [:args, :attempt, :id, :max_attempts, :meta, :queue, :tags, :worker]),
-        integration_meta: %{oban: %{job: job}}
-      )
+      if is_struct(exception) do
```

Comment:
You may want more specifically [`is_exception`](https://hexdocs.pm/elixir/1.12.3/Kernel.html#is_exception/1) here, rather than just `is_struct`, because I think `Exception.message/1` below won't work on just any general struct.

This may not really matter in practice, though, because I think the errors will probably never end up being a struct other than an Exception.

Reply:
Probably not but good to use the right tool for the job. Thanks and updated!